src/lib/server.ml

```diff
@@ -608,7 +612,7 @@ let handle_request ~server_state ~body req : (answer, _) Deferred_result.t =
     wrong_request "Wrong path" other
 
 
-(** {2 Start/Stop The Server} *)
+(** {3 Start/Stop The Server} *)
```

Comment:
the number in ocamldoc's `{<n> ...}` syntax is the level of the heading (like `h1`, `h2`, ... in HTML) both titles should be at the same level.